### PR TITLE
repairs to early return for sendNackPackets

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -2693,7 +2693,7 @@ int Application::sendNackPackets() {
             // if there are octree packets from this node that are waiting to be processed,
             // don't send a NACK since the missing packets may be among those waiting packets.
             if (_octreeProcessor.hasPacketsToProcessFrom(nodeUUID)) {
-                packetsSent = 0;
+                return;
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: don't you want return 0; here?

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -2701,16 +2701,16 @@ int Application::sendNackPackets() {
             // retreive octree scene stats of this node
             if (_octreeServerSceneStats.find(nodeUUID) == _octreeServerSceneStats.end()) {
                 _octreeSceneStatsLock.unlock();
-                packetsSent = 0;
+                return;
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: same here

File: interface/src/Application.cpp (hunk 1 above)
Comment (birarda) [inline interface/src/Application.cpp]: the return returns from the lambda not from the function as a whole

File: interface/src/Application.cpp (hunk 2 above)
Comment (birarda) [inline interface/src/Application.cpp]: same here

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9fb8b553596e0a2111d472d10631cd34161af6ed available at: https://gist.github.com/450f0f37433541f580b6<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5375.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5375.dmg

**Pull request merged**